se/se_epub_lint.py

```diff
@@ -175,6 +175,7 @@
 "m-070", "Glossary entries not present in the text:"
 "m-071", "DP link must be exactly [text]The Online Distributed Proofreading Team[/]."
 "m-072", "DP OLS link must be exactly [text]Distributed Proofreaders Open Library System[/]."
+"m-073", "Images found in ebook, but no [attr]role[/] property set to [val]wat[/] in metadata for the writer of the alt text."
```

Comment:
Can you use one of the unused codes below instead of adding a new one?

Reply:
Done, hadn’t realised they were up for grabs. (`m-040` is actually in sequence with the other image accessibility metadata linting rules.)